docs/typescript/logging.md

```diff
@@ -13,33 +13,71 @@ Activities run in the standard Node.js environment and can use any Node.js logge
 
 Logging from Workflows is tricky for 2 reasons:
 
-1. Workflows run in an isolated JS environment and may not do any I/O
-1. Workflow code might get replayed generating duplicate log messages
+1. Workflows run in a sandboxed environment and may not do any I/O
```

Comment:
this isnt something i feel strongly about but always felt that "sandbox" might be a friendlier term than "isolated" or "deterministic". wdyt? happy to revert